### PR TITLE
docs(README): align packages

### PR DESCRIPTION
**Pull request by irmerk**
# No Issue
Align documentation with other repos

### Changes
- Change PR template to point to closing an issue
- Change Issue templates to align with other AP GH repos
- Remove ecosystem to `README`
- Add `README` to `dingus`
- Update repo structure in `README` for `markdown-transform`
- Clarify `markdown-it-...` repository `README`s

### Flags
- N/A

### CODE REVIEW

File: packages/dingus/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TemplateMark Dingus
+
+An online tool which lets you edit the markdown and see it rendered as HTML, or as a document object model. This is [deployed on Netlify at this link](https://templatemark-dingus.netlify.app/).
+
+Further documentation can be found at [this link](https://docs.accordproject.org/docs/markup-preliminaries.html#dingus).
+
+## Installation
```
Comment (jeromesimeon) [inline packages/dingus/README.md:7]: I would remove `usage` which may not make sense.. (it's just a demo / testing thing).

In `Installation` (maybe better named `run locally`) :

```
npm run dingus
open demo/index.html
```

File: packages/markdown-it-cicero/README.md [hunk 2]
```diff
@@ -1,30 +1,17 @@
-# CiceroMark Transform
+# Markdown-it for CiceroMark
```
Comment (jeromesimeon) [inline packages/markdown-it-cicero/README.md:1]: `Markdown-it plugin`

File: packages/markdown-it-cicero/README.md [hunk 3]
```diff
@@ -1,30 +1,17 @@
...
-These are expressed using markdown code blocks and html inlines to ensure that they are safely persisted within markdown text.
-
-Use `CiceroMarkTransform` to map from the CommonMark DOM nodes to CiceroMark DOM nodes.
-
 ## Installation
 
-```
-npm install @accordproject/markdown-cicero --save
-```
+In progress...
```
Comment (jeromesimeon) [inline packages/markdown-it-cicero/README.md:10]:
```
npm install @accordproject/markdown-it-cicero
```

File: packages/markdown-it-template/README.md [hunk 4]
```diff
@@ -1,30 +1,17 @@
-# CiceroMark Transform
+# Markdown-it for TemplateMark
```
Comment (jeromesimeon) [inline packages/markdown-it-template/README.md:1]: `Markdown-it plugin`

File: packages/markdown-it-template/README.md [hunk 5]
```diff
@@ -1,30 +1,17 @@
...
-These are expressed using markdown code blocks and html inlines to ensure that they are safely persisted within markdown text.
-
-Use `CiceroMarkTransform` to map from the CommonMark DOM nodes to CiceroMark DOM nodes.
-
 ## Installation
 
-```
-npm install @accordproject/markdown-cicero --save
-```
+In progress...
```
Comment (jeromesimeon) [inline packages/markdown-it-template/README.md:10]:
```
npm install @accordproject/markdown-it-template --save
```

**Review by jeromesimeon — CHANGES_REQUESTED** [review]
Nice! A few suggestions.

**Comment by irmerk** [comment]
Applied those suggestions @jeromesimeon

**Review by jeromesimeon — APPROVED** [review]
LKTM

**Pull request merged**